crates/matrix-sdk-crypto/src/machine.rs

```diff
@@ -1611,12 +1611,14 @@ impl OlmMachine {
         timeout: Option<Duration>,
     ) -> StoreResult<()> {
         if let Some(timeout) = timeout {
+            tracing::warn!("WAITING");
```

Comment:
Why not import the macro?

Reply:
oh i'm just using that for debugging purposes and will remove it thereafter